src/collections/_documentation/development/sdk-dev/event-payloads/contexts.md

```diff
@@ -298,6 +298,10 @@ Example:
 
 ## Examples
 
+The following example illustrates the contexts part of the [event payload]({%-
+link _documentation/development/sdk-dev/event-payloads/index.md -%}) and omits
+other attributes for simplicity.
```

Comment:
Took the same wording as we have in the Breadcrumbs page, after replacing s/breadcrumbs/contexts.